batchflow/models/tf/base.py

```diff
@@ -1916,7 +1888,7 @@ def get_num_channels(self, tensor, **kwargs):
 
     @classmethod
     def num_channels(cls, tensor, data_format='channels_last'):
-        """ Return number of channels in the input tensor
+        """ Return number of channels in the input tensor.
```

Comment:
ok, what's the difference between `get_num_channels` and `num_channels`? Can we leave only one method?

Reply:
All of the `get_###` methods rely on `tensor_config`, which is created and stored for some tensors during `_make_inputs`. That allows to better control complex networks with multiple inputs of different `data_formats`, for example